pom.xml

```diff
@@ -336,12 +335,10 @@
     </profile>
     
     <!-- using github.com/google/error-prone-javac is required when running on JDK 8 -->
+    <!-- see instructions at https://errorprone.info/docs/installation -->
     <profile>
       <id>errorprone8</id>
       <activation>
-        <property>
```

Comment:
Not acceptable, the full QA build must activate when the "-Dqa" parameter is provided on the command line, as per documentation. This includes errorprone.

Reply:
See also: https://github.com/geotools/geotools/blob/master/docs/developer/conventions/code/qa.rst